cmd/build/build.sh

```
@@ -519,22 +525,37 @@ goimports -w .
-echo "Building worker binary..."
+# URL-rewrite ldflag goes onto whichever binary actually links the
+# BAML call path. Subprocess builds bake it into cmd/worker;
+# inprocess builds bake it into cmd/serve because cmd/worker is not
+# built at all.
 WORKER_LDFLAGS=""
+SERVE_LDFLAGS=""
 if [ -n "${BAML_REST_BASE_URL_REWRITES:-}" ]; then
     echo "Baking in base URL rewrites: ${BAML_REST_BASE_URL_REWRITES}"
-    WORKER_LDFLAGS="-X 'github.com/invakid404/baml-rest/bamlutils/urlrewrite.builtinRules=${BAML_REST_BASE_URL_REWRITES}'"
+    if [ "${INPROCESS:-false}" = "true" ]; then
+        SERVE_LDFLAGS="-X 'github.com/invakid404/baml-rest/bamlutils/urlrewrite.builtinRules=${BAML_REST_BASE_URL_REWRITES}'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)